pgbot/utils.py

```diff
@@ -389,3 +388,58 @@ def code_block(string: str, max_characters=2048):
         return f"```\n{string[:max_characters - 7]} ...```"
     else:
         return f"```\n{string[:max_characters]}```"
+
+
+def remove_all(list_thing, thing_to_remove):
+    copy = list_thing[:]
+    for _ in range(list_thing.count(thing_to_remove)):
+        copy.remove(thing_to_remove)
+    return copy
+
+
+def return_insert(list_to_insert, index, obj):
+    copy = list_to_insert[:]
+    copy.insert(index, obj)
+    return copy
+
+
+async def filter_user(user_context, guild):
```

Comment:
You can probably get rid of this function, because it looks like it is a bit buggy, and after doing the argument updates, you will not need this function too